srcpkgs/pantalaimon/template

```diff
@@ -8,10 +8,10 @@ depends="python3-attrs python3-aiohttp python3-appdirs python3-click
  python3-keyring python3-logbook python3-peewee python3-janus
  python3-cachetools python3-prompt_toolkit python3-matrix-nio
  python3-dbus python3-gobject python3-pydbus python3-notify2"
-checkdepends="${depends}"
 short_desc="Proxy daemon for matrix.org clients supporting end-to-end encryption"
 maintainer="Adam Beckmeyer <adam_gpg@thebeckmeyers.xyz>"
 license="Apache-2.0"
 homepage="https://github.com/matrix-org/pantalaimon"
-distfiles="${homepage}/archive/${version}.tar.gz"
+distfiles="https://github.com/matrix-org/pantalaimon/archive/refs/tags/${version}.tar.gz"
 checksum=82437a6265a3c747618b422d55138181e799da906111e02e7577853b33f5433b
+make_check=no # requires multiple unpackaged dependencies
```

Comment:
could you mention at least some dependencies that are not packaged? If we package them, we can notice these tests can be enabled with a simple grep.

Reply:
"fixed" by updating it to the latest version, doesn't have any reverse dependency to check for